src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -60,6 +60,142 @@ For more information on how to configure your device for test and setup test use
 
 _Note_ that, as is also described on the website, using test BankID users does require a reconfiguration of the BankID application. This means it cannot be used for real BankID. So if you are Swedish and already have BankID on your phone, you may want to use a spare phone for testing.
 
+## BankID Phone Authentication
+
+<a href="https://www.bankid.com/en/foretag/telephone-identification" target="_blank">BankID in telephone calls</a> facilitates user authentication during a phone call.
+
+### Overview
+
+The Phone authentication flow is designed for scenarios where a customer is talking with your customer service representative over the phone, and you need to verify that the caller is who they say they are. 
```

Comment:
`where a user` might be better to avoid the double customer:
![image](https://github.com/criipto/docs/assets/699049/18cd65ae-c98e-4b36-bd0b-f414b20a4fd1)

Reply:
Good point, the CIBA spec provides more examples than just a phone call. But the Swedish BankID only mentions phone calls [here](https://www.bankid.com/en/foretag/telephone-identification).  
Does it also work for chats? @kasperhj